### ISSUE TITLE
model.npz Share

### ISSUE THREAD

**Issue by Ayawang** [facebookresearch/AGRoL#21]
Today I encountered a problem where I couldn't find the model.npz file. Since this is my first time working on such a project, after some searching, I finally found the relevant resources. Now, I'd like to share them with everyone, hoping it might be helpful. Best wishes for everyone’s work!

Download link for the SMPL model: If you need to download the SMPL model files, you can do so through the following link: [SMPL Official Download Page](https://smpl.is.tue.mpg.de/download.php).

Download link for DMPLs model: For those who need DMPLs (Dynamics Models for People in Loose Clothing), you can download them via this link: [MANO Official Download Page](https://mano.is.tue.mpg.de/download.php).

Both of these are needed and should be downloaded.

**Comment by bring-nirachornkul** [comment]
Yep, they failed to mention this in the README.md. I already push these links to their repo. waiting for them to accept it.

**Comment by dulucas** [comment]
> Yep, they failed to mention this in the README.md. I already push these links to their repo. waiting for them to accept it.

Accepted
